native/tsconfig.json

```diff
@@ -20,7 +18,19 @@
     "typeRoots": ["./src/@types", "./node_modules/@types", "../node_modules/@types"],
     "types": ["jest", "node", "react-native", "styled-components-react-native"]
   },
-  "include": ["src/**/*", "./jest.setup.ts", "index.ts", "jest.config.ts", "metro.config.ci.js", "metro.config.js"],
+  "include": [
+    "src/**/*",
+    "jest.setup.ts",
+    "index.ts",
+    "jest.config.ts",
+    "babel.config.js",
+    "metro.config.ci.js",
+    "metro.config.js",
+    ".yarn/plugins/root-command.js",
+    "react-native.config.js",
+    "run.js",
+    "src/assets/licenses.json"
```

Comment:
Why do we need to explicitly include that? First, it is json and second it should be included with `src/**/*` already, shouldn't it?